### PR TITLE
Added LOGO In Readme File

### PR DESCRIPTION
**Pull request by Ananthavijay**
Fixes : https://github.com/CircuitVerse/Interactive-Book/issues/96
- This is done as per my google code in task

### CODE REVIEW

**Comment by deepak2431** [comment]
@Ananthavijay Looks good to me. 
Thanks!

**Comment by deepak2431** [comment]
@Ananthavijay What do you think of adding the logo below the title? I think it would look much better.

**Comment by Ananthavijay** [comment]
Oh ok bro .. Just a Second

**Comment by Ananthavijay** [comment]
Done @deepak2431 
Thanks

File: README.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
 
 [Join Mailing List](https://circuitverse.us20.list-manage.com/subscribe?u=89207abda49deef3ba56f1411&id=29473194d6)
 
+
+![download](https://user-images.githubusercontent.com/35162705/69917985-7f323900-1492-11ea-994e-6c2bdc999cc0.png)
```
Comment (deepak2431) [inline README.md]: Add it just below the title "Interactive Book".

**Comment by Ananthavijay** [comment]
Done @deepak2431 
Sorry for making this simple one big

**Comment by deepak2431** [comment]
@Ananthavijay It's okay no problem. Thanks!

**Pull request merged**